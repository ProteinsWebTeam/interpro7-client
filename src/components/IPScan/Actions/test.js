// @flow
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import { Actions } from '.';

const renderer = new ShallowRenderer();

describe('<Actions />', () => {
  test('should render', () => {
    renderer.render(
      <Actions
        localID={'internal-1560936888649-1'}
        jobs={{
          'internal-1560936888649-1': {
            metaData: {
              localID: 'internal-1560936888649-1',
              localTitle: null,
              remoteID: null,
              saved: false,
              status: 'created',
              times: {
                created: 1560936888650,
                lastUpdate: 1560936888650,
              },
              type: 'InterProScan',
            },
          },
        }}
        deleteJob={() => {}}
        goToCustomLocation={() => {}}
        updateJob={() => {}}
        withTitle={false}
      />,
    );
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
