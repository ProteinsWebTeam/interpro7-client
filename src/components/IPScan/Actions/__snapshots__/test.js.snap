// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Actions /> should render 1`] = `
<nav
  className="buttons"
>
  <Tooltip
    title={
      <div>
        <b>
          Delete job
        </b>
        : This remove this job from your browser. Remember that job's data is only kept in our servers for 7 days
      </div>
    }
  >
    <button
      aria-label="Delete job"
      className="icon icon-common ico-neutral"
      data-icon="ï‡¸"
      onClick={[Function]}
    />
  </Tooltip>
  <Tooltip
    title={
      <div>
        <b>
          Save in Browser
        </b>
        : If you save the data of this job in your browser, you will be able to display it here even if it gets deleted in our servers or you lost internet connection.
      </div>
    }
  >
    <button
      aria-label="Save in Browser"
      className="icon icon-common ico-neutral"
      data-icon="S"
      onClick={[Function]}
    />
  </Tooltip>
</nav>
`;
